packages/esm-patient-appointments-app/src/appointments/appointments.resource.tsx

```diff
@@ -1,5 +1,62 @@
+import dayjs from 'dayjs';
+import useSWR from 'swr';
 import { openmrsFetch } from '@openmrs/esm-framework';
-import { Appointment, AppointmentPayload } from '../types';
+import { AppointmentPayload, AppointmentService, AppointmentsFetchResponse } from '../types';
+
+export const appointmentsSearchUrl = `/ws/rest/v1/appointments/search`;
+
+export function useAppointments(patientUuid: string, startDate: string, abortController: AbortController) {
+  /* 
+    SWR isn't meant to make POST requests for data fetching. This is a consequence of the API only exposing this resource via POST.
```

Comment:
Why... is the API like that...???

Reply:
This seems like a good workaround, anyway, assuming it's a fake POST (doesn't actually do any mutation)

Reply:
But yeah, who do we talk to to fix that

Reply:
I don't know, I think I fielded this same question on a squad call in the past even.

Reply:
Who owns that API?